AutoRest/Generators/CSharp/Azure.CSharp.Tests/Expected/SwaggerBat/SubscriptionIdApiVersion.Cs/MicrosoftAzureTestUrlExtensions.cs

```diff
@@ -0,0 +1,15 @@
+namespace Fixtures.Azure.SwaggerBatSubscriptionIdApiVersion
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Rest;
+    using Microsoft.Azure;
+    using Models;
+
+    public static partial class MicrosoftAzureTestUrlExtensions
```

Comment:
We should file a bug for this - we don't need the extensions class if there are no operations on the client